### ISSUE TITLE
Add the ability to Manage Offers

### ISSUE THREAD

**Issue by MikeFair** [tolitius/stellar-mc#4]
I noticed the ability to manage offers on the DEX was missing.
This may or may not apply to allowing path payments; they are rather complicated to submit.

**Comment by tolitius** [comment]
yes, this is definitely something we should add. the reason it's still not there, it is a bit more involved than simple(r) operations, an I want to feel comfortable with APIs first to avoid unnecessary tech debt.

now, as we worked through #2, it made some improvements to the API, and we might be ready to increase complexity to tackle offers and path payments.

**Comment by tolitius** [comment]
note to self, will makes sense to tackle these 3 as separate tasks/issues:

* Manage Offer
* Create Passive Offer
* Path Payment

is there an order book that has a testnet based UI? i.e. there are http://testnet.stellarchain.io and https://stellar.expert/explorer/testnet/ but an order book view would really be helpful developing support for these 3 operations
